docs/DevGuide/DomainConcepts.md

```diff
@@ -4,61 +4,94 @@ See LICENSE.txt for details.
 \endcond
 # Domain Concepts {#domain_concepts}
 
-* Logical Coordinates:<br>
-  The coordinate frame of a reference cell.
+* Inertial Coordinate Frame:<br>
+  A global coordinate frame covering the computational domain that is also the
+  coordinate frame in which the initial (boundary) value problem that is being
+  solved is specified.  Denoted by Frame::Inertial.
 
-* Domain:<br>
-  The computational domain is partitioned into cells called Blocks. Furthermore, each Block
-  must have at most one neighboring Block in each Direction.
+* Logical Coordinate Frame:<br>
+  The coordinate frame of a reference cell.  In the cell the logical coordinates
+  cover the interval \f$[-1, 1]\f$ in each dimension.
+
+* CoordinateMap "Coordinate Map":<br>
+  A mapping between two coordinate frames.  Coordinate maps are allowed to be
+  time-dependent, as long as the time coordinate itself is unchanged.
+
+* Direction:<br>
+  A logical coordinate axis and a label "Upper" or "Lower" depending on whether
+  the Direction is aligned with or anti-aligned with the logical coordinate
+  axis, respectively.
 
 * Block:<br>
-  A reference cell embedded into a subset of the computational domain. Blocks
-  are identified by unique integral values.
+  The computational domain is partitioned into a set of non-overlapping,
```

Comment:
Other uses of `computational domain` that I've seen refers to the undistorted cells, before the coordinate mapping is applied (which maps points from the computational domain to the "physical domain")

Maybe
```
The domain is partitioned into a set of non-overlapping
hexahedral regions called Blocks. Each Block must have at most one
neighboring Block in each of its Directions. The region is mapped from
the computational domain (where the region is the reference cell) to the
physical domain using a Coordinate Map from the...

Reply:
I agree with what Larry is saying. The computational domain strictly as the region covered by the simulation. Nothing to do with frames, blocks, elements, or anything. That all is how you partition the computational domain. Blocks are used to partition the computational domain into reference cells that have (currently) logically Cartesian axes. A coordinate map is used to partition the computational domain into non-overlapping (and possibly curved) cells. The computational domain is not in the reference cell, it's the "physical" or "inertial" domain.

Reply:
I'm fine with us using "computational domain" and "physical domain" in the way that you have defined them, but that is not the way I've seen those terms used in e.g. CFD papers, at least, from what I am seeing [here](https://www.google.com/search?q=computational+domain+vs+physical+domain&source=lnms&tbm=isch&sa=X&ved=2ahUKEwih_96mn7TzAhXGCjQIHQfBB04Q_AUoAXoECAEQAw&biw=1004&bih=979&dpr=1#imgrc=wLLL0_3YDGz71M&imgdii=2iZ84hGpi6I9zM) - the "computational domain" refers to the (often) cartesian coordinate system which is then mapped onto the "physical domain." I am not strictly opposed to our use of "computational domain," I just wanted to make note that the "computational domain," as we use it, is not the logical frame coordinate system, while in the CFD literature (to the best of my knowledge) it is.

Maybe I should have clarified that the "Other uses of `computational domain` that I've seen" are outside of SpECTRE - within SpECTRE I have not noticed an inconsistent use of terms, which is why I'm not strictly opposed to the use. (But it is, as far as I can tell, not in agreement with the terminology used in CFD, and might confuse people who were not aware of that) We might have already agreed to not strictly follow their terminology/ conventions, so this could be a non-issue - I just wanted to make clear where I was coming from.

Reply:
From telecon: I'm okay with either computational domain meaning what CFD folks say or what we say. I agree we should make a note if we differ just to be clear :)